include/sentry.h

```diff
@@ -1186,6 +1186,26 @@ SENTRY_API void sentry_options_add_attachment(
 SENTRY_API void sentry_options_add_attachment_n(
     sentry_options_t *opts, const char *path, size_t path_len);
 
+/**
+ * The attachment_type.
+ */
+typedef enum {
+    ATTACHMENT,
+    MINIDUMP,
+    VIEW_HIERARCHY,
+} sentry_attachment_type_t;
```

Comment:
Because we no longer provide an API that needs to expose the attachment types, maybe we should move this to where it is actually used?